source/libtomo/recon/c/mlem.c

```diff
@@ -47,20 +47,12 @@
 
 void
 mlem(const float* data, int dy, int dt, int dx, const float* center, const float* theta,
-     float* recon, int ngridx, int ngridy, int num_iter, int accel, int pool_size,
+     float* recon, int ngridx, int ngridy, int num_iter, int pool_size,
      const char* interp, const char* device, int* grid_size, int* block_size)
```

Comment:
There are additional parameters such as `pool_size` which are no longer needed in `mlem.c`. Please remove these.